### PR TITLE
implemented to lower case feature

### PR DESCRIPTION
**Pull request by SskYwaLkeR**

### CODE REVIEW

File: implementations/toLowerCase.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  if(value >= 65 && value <= 90){
+    //convert it to lowercase
+    temp += String.fromCharCode(value + 32);
+  }else{
+    //else add the original character
+    temp += chars;
+  }
+}
+    
+}
```
Comment (ashokdey) [inline implementations/toLowerCase.js]: Breaking for the following case:

```javascript
const a = '-21Hello 0 World @123';
```

Comment (SskYwaLkeR) [reply implementations/toLowerCase.js]: Hi sorry I'm new here I refactored the code can you tell me the new PR is submitted if yes then have a look into that

Comment (ashokdey) [reply implementations/toLowerCase.js]: You can have a look at #7 As of now I have to close this PR as it is still incorrect and untidy.

File: implementations/toLowerCase.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    //convert it to lowercase
+    temp += String.fromCharCode(value + 32);
+  }else{
+    //else add the original character
+    temp += chars;
+  }
+}
+    console.log(temp)
+    return temp;
+}
```
Comment (ashokdey) [inline implementations/toLowerCase.js]: Although the implementation seems correct, it is not following the ESLint guidelines. Also, I discourage the use of `for-of`

There should not be any `console.log()` in the final code.

**Comment by ashokdey** [comment]
closing as #7 closes this before your PR.

**Pull request closed without merging**